build/README.md

```diff
@@ -0,0 +1,7 @@
+These directories contain Dockerfiles for the Stackdriver Logging agent build
```

Comment:
We should add something along the lines of, "These images are built on top base Linux distro base images, because Omnibus configures the build based on the detected distribution." For me, that was the key insight from our conversation.

Reply:
Done, PTAL.

Reply:
LG, thanks!